mesheryctl/internal/cli/root/system/config.go

```diff
@@ -329,7 +329,16 @@ var configCmd = &cobra.Command{
 	Use:   "config",
 	Short: "Configure Meshery",
 	Long:  `Configure the Kubernetes cluster used by Meshery.`,
-	Args:  cobra.ExactArgs(1),
+	Args: func(_ *cobra.Command, args []string) error {
+		const errMsg = `Usage: mesheryctl system config [aks|eks|gke|minikube]
+Example: mesheryctl system config eks
+Description: Configure the Kubernetes cluster used by Meshery.`
+
+		if len(args) == 0 {
+			return fmt.Errorf("accepts single argument, received %d\n\n%v", len(args), errMsg)
+		}
+		return nil
+	},
```

Comment:
I think we should also handle the case for more that 1 command here. Is this case being handled or is there an error in my understanding?

Reply:
We are handling error when ```noArgs``` is passed in the above statement.

Reply:
@hershd23 Can you brief about what issues are you facing with ```config``` command !

Reply:
I think he probably means that the case when there are more than 1 args should also be handled.

Reply:
@nishantwrp Any Suggestions over this 
```
mesheryctl system config eks --token auth.json sad
Error: too many arguments
```

Reply:
That works, maybe something like "More than one config name provided" would be better?

Reply:
Nice